src/Component.php

```diff
@@ -37,38 +39,32 @@ private function processDir(Finder $finder, string $dir, string $filter) : void
             } else {
                 $destinationDir = $this->getDataDir() . "/out/$dir/" . $inFile->getRelativePath();
                 $fs->mkdir($destinationDir);
-                $destinationFile = $destinationDir . '/' . $inFile->getFilename();
-                $this->processFile($inFile, $destinationFile, $filter);
+                $destinationFile = $destinationDir . "/" . $inFile->getFilename();
+                $this->processFile($inFile, $destinationFile, $config);
             }
         }
     }
 
-    private function processFile(SplFileInfo $inFile, string $destinationFileName, string $filter) : void
+    private function processFile(SplFileInfo $inFile, string $destinationFileName, Config $config) : void
     {
-        try {
-            $source = fopen($inFile->getPathname(), 'r');
-            $destination = fopen($destinationFileName, 'w');
-        } /** @noinspection PhpRedundantCatchClauseInspection */ catch (\ErrorException $e) {
-            throw new \Exception('Failed opening in/out ' . $inFile->getPathname() . ' ' . $e->getMessage());
+        $command = "iconv";
+        if ($config->ignoreErrors()) {
+            $command .= " -c";
         }
+        $command .= " --from-code=" . $config->getSourceEncoding();
```

Comment:
nemělo by se to oescapovat?

Reply:
jde tam jenom uzavřenej set hodnot, ale pro sichr teda jo. nicméně cesty, co jsou na řádku pod tím, oescapovaný bejt musej, dík

✅